.github/workflows/main.yml

```diff
@@ -3,8 +3,10 @@ on:
   push:
     branches:
       - master
-    pull_request:
-      types: [opened, reopened, synchronize]
+  pull_request:
```

Comment:
🤦 Sorry about that

Reply:
No worries. I reviewed it and I missed it, too :D